src/components/pages/StudentsPage/WelcomeMessage.css.tsx

```diff
@@ -0,0 +1,18 @@
+import { css } from '@emotion/react';
+
+const unpairedMessageContainer = css`
+  background: #f8e5e0;
+  color: red;
+  width: 98%;
+  max-width: 800px;
+  border: 1px solid white;
+  border-radius: 10px;
+  padding: 70px 0;
+  text-align: center;
+`;
+
+const unpairedMessageCSS = {
```

Comment:
For consistency, lets use the same variable name as the fiename.
```suggestion
const welcomeMessageCSS = {
```